pkg/service/backup/README.md

```diff
@@ -0,0 +1,60 @@
+# Backup service
+
+### Useful links
+- official documentation of what the backup in scylla-manager is - https://manager.docs.scylladb.com/stable/backup/index.html
+- `sctool backup` command - https://manager.docs.scylladb.com/stable/sctool/backup.html
+- scylla manager agent config file (look for `Backup general configuration.`) - https://manager.docs.scylladb.com/stable/config/scylla-manager-agent-config.html
+- schylla manager config file (look for `Backup service configuration`) - https://manager.docs.scylladb.com/stable/config/scylla-manager-config.html
+
+### General picture 
+
+#### Scheduling the backup ####
+
+![Scheduling the backup](scylla-manager-backup.drawio.svg)
+
+Backup service is responsible for handling backup tasks scheduled either via Scylla Manager REST API or via sctool (CLI client for REST API).
+
+#### Performing the backup ####
+
+```
```

Comment:
you can annotate code blocks with `go` keyword to enable syntax highlighting like so https://www.markdownguide.org/extended-syntax/#syntax-highlighting

This applies to all go snippets in this PR